k2/csrc/moderngpu_context.cu

```diff
@@ -0,0 +1,50 @@
+/**
+ * @brief A context for moderngpu with a better memory allocator.
+ *
+ *
+ * @copyright
+ * Copyright (c)  2020  Mobvoi AI Lab, Beijing, China (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include "k2/csrc/context.h"
+#include "k2/csrc/moderngpu_context.h"
+#include "moderngpu/context.hxx"
+
+namespace {
+
+class ModernGpuContext : public mgpu::standard_context_t {
```

Comment:
i wonder if this should be called ModernGpuAllocator to clarify that it's not an instance of k2::Context.

Reply:
Fixed.